frontend/src/util/download.ts

```diff
@@ -26,17 +39,42 @@ export const download = (
 
 /** download table data as csv */
 export const downloadCsv = (data: unknown[], filename: string | string[]) =>
-  download(stringify(data), filename, "text/csv;charset=utf-8", "csv");
+  download(
+    stringify(data, { header: true }),
+    filename,
+    "text/csv;charset=utf-8",
+    "csv",
+  );
+
+/** download table data as tsv */
+export const downloadTsv = (data: unknown[], filename: string | string[]) =>
```

Comment:
currently, our network script only carries three columns
- node1, node2, direction
This info is used to calculate node freq, edge freq, auto-loops, etc. currently, there's no additional metadata. If a particular input format is required for this cytoscape network, then we need to make sure the molevolvr outputs return the required values.

Reply:
I wanted to make sure arbitrary data download was supported by these util funcs.

There shouldn't be any bending-over-backwards to fit Cytoscape. The way I try to write these components is to take a logical, general purpose set of inputs (e.g. "type" rather than "species", or "strength" rather than "p-value"), and translate that to whatever the library we're using needs. So later if we want to switch off of Cytoscape to something else, we need only change this one component.

I think this warrants some discussion in our next meeting, and will also become clearer as we start to hook up real endpoints.